docs/source/conf.py

```diff
@@ -50,7 +50,7 @@
 ########################################
 
 cminx_out_dir = os.path.join(src_dir, "developer", "cmake_test")
-cminx_in_dir = os.path.join(root_path, "cmake_test")
+cminx_in_dir = os.path.join(root_path, "cmake/cmake_test")
```

Comment:
We should let os.path.join determine the path separators here:
```suggestion
cminx_in_dir = os.path.join(root_path, "cmake", "cmake_test")
```